design-documents/graph-ql/coverage/customer/SubscribeEmailToNewsletter.graphqls

```diff
@@ -3,7 +3,7 @@ type Mutation {
 }
 
 type SubscribeEmailToNewsletterOutput {
-    status: SubscriptionStatusesEnum @doc(description: "Returns a status of subscription")
+    status: SubscriptionStatusesEnum @doc(description: "Returns a status of newsletter subscription")
 }
 
 enum SubscriptionStatusesEnum {
```

Comment:
To better align with the existing enum values,   change "NOT_ACTIVE"  status to INACTIVE

Reply:
yes but we can't remove that value
 we have to keep it, add a comment that's not used and add "INACTIVE" as a new value